src/main/webapp/app/exercises/shared/example-submission/example-submissions.component.ts

```diff
@@ -36,6 +37,13 @@ export class ExampleSubmissionsComponent implements OnInit, OnDestroy {
             this.accountService.setAccessRightsForCourse(exercise.course);
             this.exercise = exercise;
         });
+        if (this.exercise && this.exercise.exampleSubmissions && this.exercise.exampleSubmissions.length > 0) {
+            this.exercise.exampleSubmissions.forEach((exampleSubmission) => {
```

Comment:
[nit] would `this.exercise?.exampleSubmissions?.forEach((exampleSubmission) => { ... });` work?
if so, then we don't need the extra if block

Reply:
same as previous comment, I'll check